src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -145,35 +145,37 @@ def execute(self):
         # start threads in MSManager which should call this method
         summary = dict(UNMERGED_REPORT)
 
-        # refresh statistics on every poling cycle
-
+        # fetch the protectedLFNs list
         try:
-            self.rseConsStats = self.rucioConMon.getRSEStats()
-            # self.logger.debug("RucioConMon Stats: %s", pformat(self.rseConsStats))
             self.protectedLFNs = set(self.wmstatsSvc.getProtectedLFNs())
             # self.logger.debug("protectedLFNs: %s", pformat(self.protectedLFNs))
 
             if not self.protectedLFNs:
-                msg = "Could not fetch list with protectedLFNs from WMStatServer. "
+                msg = "Could not fetch the protectedLFNs list from WMStatServer. "
                 msg += "Skipping the current run."
-                msg += "\nTotal number of RSEs processed: %s."
-                msg += "\nTotal number of files fetched from RucioConMon: %s."
-                msg += "\nNumber of RSEs cleaned: %s."
-                msg += "\nNumber of files deleted: %s."
-                self.logger.info(msg,
-                                 0,
-                                 0,
-                                 0,
-                                 0)
-                self.updateReportDict(summary, "total_num_rses", 0)
-                self.updateReportDict(summary, "total_num_files", 0)
-                self.updateReportDict(summary, "num_rses_cleaned", 0)
-                self.updateReportDict(summary, "num_files_deleted", 0)
+                self.logger.info(msg)
```

Comment:
I'd suggest to set it to error, since it causes the cycle execution to be skipped.